21-Hello-World/06-Type-Level-Programming/ReadMe.md

```diff
@@ -33,6 +33,7 @@ elemAtIndex :: forall a n. HasElemAtIndex n => n -> IndexedArray n a
 elemAtIndex index array = -- implementation
 
 elemAtIndex 3 (IndexedArray 3 ["a", "b", "c", "d"]) -- "d"
+elemAtIndex 3 (IndexedArray 3 ["a", "b"]) -- compiler error!
```

Comment:
Perhaps to make this more obvious, the elements of the array should be changed from String to Int?
```purescript
elemAtIndex 3 (IndexedArray 3 [0, 1, 2, 3] -- 3
elemAtIndex 3 (IndexedArray 1 [0, 1] -- compiler error!
```